[YAML] Add note about running yaml pipelines directly via gcloud. (#30381)
apache/beam: website/www/site/content/en/documentation/sdks/yaml.md

(This requires [Graphviz](https://graphviz.org/download/) to be installed to render the pipeline.)

> A yaml pipeline can also be directly submitted Dataflow via

Reviewer: ```suggestion
You can also submit a YAML pipeline directly by using the Dataflow CLI command
```

> [gcloud](https://cloud.google.com/sdk/gcloud/reference/beta/dataflow/yaml/run),

Reviewer: ```suggestion
[`gcloud beta dataflow yaml run`](https://cloud.google.com/sdk/gcloud/reference/beta/dataflow/yaml/run).
```

> no local installation of the Beam SDKs required.

Reviewer: ```suggestion
When you use the `gcloud` CLI, you don't need to install the Beam SDKs locally.
```
